releasenotes/notes/sabre-ctrl-flow-o1-431cd25a19adbcdc.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
```

Comment:
This probably should be an upgrade note, since it's internal details for users of `transpile()` but it does mean that the layout and routing will produces different circuits when using control flow.